apis/v1alpha1/ack-generate-metadata.yaml

```diff
@@ -1,13 +1,13 @@
 ack_generate_info:
-  build_date: "2022-09-02T16:27:09Z"
-  build_hash: cdb0d99c7c1d3d00e71623eb56e650960dcef0d5
-  go_version: go1.18.3
-  version: v0.19.3-2-gcdb0d99
-api_directory_checksum: 83bde1f77a49ebd8acd36a1c5e231138b67aa883
+  build_date: "2022-09-06T20:20:20Z"
+  build_hash: 585f06bbd6d4cc1b738acb85901e7a009bf452c7
+  go_version: go1.18.2
+  version: v0.20.0
```

Comment:
nit: It is not visible in this case, but generating controller using non matching `runtime`/`code-generator` versions "can" cause problems. Ideally this PR should be generated using code-generator `v0.19.3`

Reply:
that's good to know!  it seems after it generated use v0.20.0, it's not able to go backward even i set runtime to v0.19.3 in local

Reply:
To generate code using `code-generator` v0.19.3, you can run `git checkout tags/v0.19.3`  in code-generator repository  (make sure that you have all tags fetched)

Reply:
thanks! updated to use v0.19.3 and regenerated the code